sunpy/instr/fermi.py

```diff
@@ -0,0 +1,408 @@
+from __future__ import absolute_import
+from __future__ import division
+
+import numpy as np
+import copy
+import math
+import urlparse
+import urllib, urllib2
+import tempfile
+import datetime
+import os
+import matplotlib.pyplot as plt
+
+#from bs4 import BeautifulSoup
+import astropy.units as u
+from astropy.utils.compat.odict import OrderedDict
+from astropy.coordinates import Angle, Longitude, Latitude
+
+from sunpy.time import parse_time, TimeRange
+from sunpy import sun
+from sunpy.io.fits import fits
+
+__all__ = ['download_weekly_pointing_file', 'get_detector_sun_angles_for_time',
+            'get_detector_sun_angles_for_date','plot_detector_sun_angles',
+             'met_to_utc']
+
+def download_weekly_pointing_file(date):
+    '''Downloads the FERMI/LAT weekly pointing file corresponding to the specified date.
+    This file contains 1 minute cadence data on the spacecraft pointing, useful for
+    calculating detector angles.
+
+    Parameters
+    ----------
+
+    date : datetime object
```

Comment:
all instances of `datetime object` should be replaced with `<backtick>datetime.datetime<backtick>` so that the autodoc links to the datetime documentation. (i.e. actually put backticks in the docstring)